src/handlers/package.js

```diff
@@ -46,3 +74,32 @@ export const verifyPackagePaymentHandler = async (req, res) => {
         res.status(500).json({ errorMessage: 'An unexpected error occured. Check server logs' });
     }
 }
+
+
+export const addScheduleHandler = async (req, res) => {
+    try {
+        const id = req.params.packageId;
+        const add = await addScheduleController(id, req.body)
+        if (add) {
+            res.status(200).json({ successMessage: "The schedule was successfully added" })
+            return;
+        }
+        res.status(404).json({ errorMessage: "A package with this id does not exist or no remaining session left or Vehicle unavailiable" })
```

Comment:
Split this error into 3 seperate errors, with different error codes (404 and 400) and error messages.